apps/web/src/components/Basenames/UsernameProfileSectionBadges/index.tsx

```diff
@@ -19,14 +18,14 @@ function BadgesLoop({
   currentWalletIsOwner?: boolean;
 }) {
   return (
-    <ul className="mb-12 mt-6 flex flex-col flex-wrap items-center gap-8 sm:flex-row sm:items-start">
+    <ul className="mt-6 grid grid-cols-2 gap-4 md:flex md:flex-row md:flex-wrap md:items-center md:gap-8">
```

Comment:
I've found sm to be larger than most (all?) the mobile viewports I've worked with. Just curious what you found the change to md got you?

Reply:
we used md in every other sections/pages for basename, that seemed to fit the design correctly here (tested on Desktop/Mobile)